nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -83,10 +83,12 @@ Costmap2DROS::Costmap2DROS(const std::string & name, tf2_ros::Buffer & tf)
   set_parameter_if_not_set("origin_y", 0.0);
   set_parameter_if_not_set("footprint", "[]");  
   set_parameter_if_not_set("footprint_padding", 0.01);
-  set_parameter_if_not_set("robot_radius", 0.1); 
+  set_parameter_if_not_set("robot_radius", 0.46);
```

Comment:
Eventually, I think the Robot should be a node and this should be a parameter of that node.